include/zephyr/pm/device_runtime.h

```diff
@@ -10,6 +10,7 @@
 
 #include <zephyr/device.h>
 #include <zephyr/kernel.h>
+#include <zephyr/pm/device.h>
```

Comment:
why is this needed?

Reply:
good catch, it's a leftover for getting `struct pm_device` when this was an inline, but then I moved it to the normal file so this can go now